pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go

```diff
@@ -360,6 +360,27 @@ func TestLeastAllocatedScoringStrategy(t *testing.T) {
 			expectedScores: []framework.NodeScore{{Name: "node1", Score: 50}, {Name: "node2", Score: 60}},
 			resources:      extendedResourceSet,
 		},
+		{
+			// Bypass the resource if the pod does not request sequentially
+			// CPU Score: ((6000 - 3000) * 100) / 6000 = 50
+			// Memory Score: ((10000 - 4000) * 100) / 10000 = 60
+			// Node1 Score: (50 * 1 + 60 * 1) / (1 + 1) = 55
+			// Node2 Score: (50 * 1 + 60 * 1) / (1 + 1) = 55
+			name: "bypass the resource if the pod does not request sequentially",
```

Comment:
sequentially? what did you mean?

although, it should probably say: if the node doesn't have a resource